lib/iex/test/iex/helpers_test.exs

```diff
@@ -419,7 +419,14 @@ defmodule IEx.HelpersTest do
     cleanup_modules([:sample])
   end
 
-  test "pid helper" do
+  test "pid/1 helper" do
+    assert "#PID<0.32767.3276>" == capture_iex(~s[pid("0.32767.3276")])
+    assert "#PID<0.5.6>" == capture_iex(~s[pid("0.5.6")])
```

Comment:
@whatyouhide I had multiple examples because those are the same examples that pid/3 had.  I wanted to keep with the convention.  Should I change pid/3's examples/tests?